docs/development/design/stack.md

```diff
@@ -0,0 +1,60 @@
+# Nexodus Stack
+
+## Summary
+
+This proposal documents the decisions made around the Nexodus software stack.
```

Comment:
I don't know how much I care, but in the `project-naming.md` design doc, we call out "Nexodus Service" as the name that refers to the control plane side. Other docs still say "Nexodus stack" or "Nexodus controller" in places, but I've been trying to make myself stick to "Nexodus Service," unless we pick something else.

You could also add here something about "early key decisions made before we started capturing things in docs/design/".